docker-compose.yml

```diff
@@ -79,35 +107,39 @@ services:
     depends_on:
       - sql-fhir
 
-  role-service:
+  #########################
+  # SPRING CLOUD SERVICES #
+  #########################
+  config-server:
     build:
       context: .
-      dockerfile: role-service/Dockerfile
+      dockerfile: config-server/Dockerfile
       args:
-        SVC: role-service
-    image: ghcr.io/ndph-arts/role-service:${GITHUB_SHA}
-    depends_on:
-      - sql-mts-data
+        SVC: config-server
     environment:
-      SERVER_PORT: "82"
-      JDBC_DRIVER: com.microsoft.sqlserver.jdbc.SQLServerDriver
-      JDBC_URL: jdbc:sqlserver://sql-mts-data:1433;databaseName=master;user=sa;password=${SAPASSWORD}
-      SPRING_CLOUD_CONFIG_ENABLED: "false"
+      SERVER_PORT: "8888"
+      SPRING_CLOUD_CONFIG_SERVER_GIT_URI: "https://github.com/NDPH-ARTS/spring-config-server.git"
```

Comment:
this url is the old one.
@darrengsmithson  feel free to delete this repo.
@tamirkamara  use
https://github.com/NDPH-ARTS/mts-trial-deployment-config.git